plat/intel/soc/stratix10/drivers/qspi/cadence_qspi.h

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __CAD_QSPI_H__
+#define __CAD_QSPI_H__
+
+#define CAD_QSPI_MICRON_N25Q_SUPPORT			1
+
+#define CAD_QSPI_OFFSET					0xff8d2000
+
+#define CAD_INVALID					-1
+#define CAD_QSPI_ERROR					-2
+
+#define CAD_QSPI_ADDR_FASTREAD				0
+#define CAD_QSPI_ADDR_FASTREAD_DUAL_IO			1
+#define CAD_QSPI_ADDR_FASTREAD_QUAD_IO			2
+#define CAT_QSPI_ADDR_SINGLE_IO				0
+#define CAT_QSPI_ADDR_DUAL_IO				1
+#define CAT_QSPI_ADDR_QUAD_IO				2
+
+#define CAD_QSPI_BANK_ADDR(x)				((x) >> 24)
+#define CAD_QSPI_BANK_ADDR_MSK				0xff000000
+
+#define CAD_QSPI_COMMAND_TIMEOUT			0x10000000
+
+#define CAD_QSPI_CFG					0x0
+#define CAD_QSPI_CFG_BAUDDIV_MSK			0xff87ffff
+#define CAD_QSPI_CFG_BAUDDIV(x)				(((x) << 19) & 0x780000)
+#define CAD_QSPI_CFG_CS_MSK				~0x3c00
+#define CAD_QSPI_CFG_CS(x)				(((x) << 11))
+#define CAD_QSPI_CFG_ENABLE				(1 << 0)
+#define CAD_QSPI_CFG_ENDMA_CLR_MSK			0xffff7fff
+#define CAD_QSPI_CFG_IDLE				(1 << 31)
+#define CAD_QSPI_CFG_SELCLKPHASE_CLR_MSK		0xfffffffb
+#define CAD_QSPI_CFG_SELCLKPOL_CLR_MSK			0xfffffffd
+
+#define CAD_QSPIDATA_OFST				0xff900000
+
+#define CAD_QSPI_DELAY					0xc
+#define CAD_QSPI_DELAY_CSSOT(x)				((x & 0xff) << 0)
+#define CAD_QSPI_DELAY_CSEOT(x)				((x & 0xff) << 8)
+#define CAD_QSPI_DELAY_CSDADS(x)			((x & 0xff) << 16)
+#define CAD_QSPI_DELAY_CSDA(x)				((x & 0xff) << 24)
+
+#define CAD_QSPI_DEVSZ					0x14
+#define CAD_QSPI_DEVSZ_ADDR_BYTES(x)			(x << 0)
+#define CAD_QSPI_DEVSZ_BYTES_PER_PAGE(x)		(x << 4)
+#define CAD_QSPI_DEVSZ_BYTES_PER_BLOCK(x)		(x << 16)
+
+#define CAD_QSPI_DEVWR					0x8
+#define CAD_QSPI_DEVRD					0x4
+#define CAD_QSPI_DEV_OPCODE(x)				((x & 0xff) << 0)
+#define CAD_QSPI_DEV_INST_TYPE(x)			((x & 0x3) << 8)
+#define CAD_QSPI_DEV_ADDR_TYPE(x)			((x & 0x3) << 12)
+#define CAD_QSPI_DEV_DATA_TYPE(x)			((x & 0x3) << 16)
+#define CAD_QSPI_DEV_MODE_BIT(x)			((x & 0x1) << 20)
+#define CAD_QSPI_DEV_DUMMY_CLK_CYCLE(x)			((x & 0xf) << 24)
```

Comment:
Add parentheses to the macro arguments. There are several occurrences more in this file.

Reply:
Thanks for the mention =D